mava/wrappers/jumanji.py

```diff
@@ -292,25 +292,27 @@ def create_agents_view(grid: chex.Array, agents_locations: chex.Array) -> chex.A
             # agents_locations: (A, 2)
 
             # Get dirty / wall tiles from first agent's obs and tile in agents dimension.
-            # (A, R, C)
-            dirty_channel = jnp.tile(grid == DIRTY, (num_agents, 1, 1))
-            wall_channel = jnp.tile(grid == WALL, (num_agents, 1, 1))
+
+            dirty_channel = jnp.tile(grid == DIRTY, (num_agents, 1, 1))  # (A, R, C)
+            wall_channel = jnp.tile(grid == WALL, (num_agents, 1, 1))  # (A, R, C)
 
             # Get each agent's position.
-            # (2, A)
-            xs, ys = agents_locations[:, 0], agents_locations[:, 1]
+            xs, ys = agents_locations[:, 0], agents_locations[:, 1]  # (A,), (A,)
 
             # Mask each agent's position so an agent can idenfity itself.
             # Sum the masked grids together for global agent information.
-            # (A, R, C)
-            pos_per_agent = jnp.repeat(jnp.zeros_like(grid)[jnp.newaxis, :, :], num_agents, axis=0)
-            pos_per_agent = pos_per_agent.at[jnp.arange(num_agents), xs, ys].set(1)
-            agents_channel = jnp.tile(jnp.sum(pos_per_agent, axis=0), (num_agents, 1, 1))
+            pos_per_agent = jnp.repeat(
+                jnp.zeros_like(grid)[jnp.newaxis, :, :], num_agents, axis=0
+            )  # (A, R, C)
```

Comment:
Sorry @RuanJohn and I disagree on this :joy:
Resolved to keep shape comments above if it doesn't cause the line to have to reformat into multiple lines
```suggestion
            # (A, R, C)
            pos_per_agent = jnp.repeat(jnp.zeros_like(grid)[jnp.newaxis, :, :], num_agents, axis=0)
```